place_tracker/lib/place_details.dart

```diff
@@ -4,12 +4,13 @@ import 'package:flutter/rendering.dart';
 import 'package:google_maps_flutter/google_maps_flutter.dart';
 
 import 'place.dart';
+import 'stub_data.dart';
 
 class PlaceDetails extends StatefulWidget {
   const PlaceDetails({
-    Key key,
     @required this.place,
     @required this.onChanged,
+    Key key,
```

Comment:
Not that it really matters, but conventionally the key argument goes first, and if there's a child or children argument, they go last.

Reply:
You-all are welcome to order parameters by whatever convention you like :-). 

If you look at the flutter sources, 
say https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/widgets/basic.dart, you'll see that we're pretty consistent.